src/components/nav.jsx

```diff
@@ -17,6 +21,21 @@ function Navbar({ isDark, toggleDarkMode, color }) {
         </div>
       </div>
       <div className="flex items-center nav-item ">
+      <div style={{marginRight: '0.5rem', marginTop: '0.5rem' }}>      
```

Comment:
- No responsive responsive/mobile friendly. 
- And logo doesn't go with the theme (a black outline twitter logo would look much better).